tests/test_events.py

```diff
@@ -104,3 +151,131 @@ def test_publish(self):
             course=block.location.course,
             run=block.location.run,
         )
+
+    @data(*[
+        ['choice_0', 'an incorrect answer'],
+        ['choice_1', 'the correct answer'],
+        ['choice_2', 'a different incorrect answer'],
+    ])
+    @unpack
+    def test_problem(self, clicked_answer_id, expected_answer_text):
+        """
+        A problem should trigger an event which is captured
+        """
+        problem = self.get_problem()
+
+        problem.handle_ajax('problem_check', {
+            "input_i4x-SGAU-SGA101-problem-"
+            "2582bbb68672426297e525b49a383eb8_2_1": clicked_answer_id
+        })
+        assert RapidResponseSubmission.objects.count() == 1
+        obj = RapidResponseSubmission.objects.first()
+        assert obj.user_id == self.instructor.id
+        assert obj.course_id == self.course.course_id
+        assert obj.problem_id.map_into_course(
+            self.course.course_id
+        ) == problem.location
+        assert obj.answer_text == expected_answer_text
+        assert obj.answer_id == clicked_answer_id
+
+    def test_double_click(self):
+        """
+        Only the last click should get captured
+        """
```

Comment:
Possibly better test name: `test_multiple_submissions`. Docstring should be updated too